From bd4ab5bba6cb378a1814c2e2acf5d15a6bba5b4c Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.xyz>
Date: Fri, 24 Feb 2017 22:47:52 +0800
Subject: [PATCH 097/105] gpio: sunxi: add compatibles for A64 pinctrl

Allwinner A64 have two pin controllers: one for generic pins and one for
"CPUs" pins.

Add compatibles for them in sunxi_gpio driver.

Signed-off-by: Icenowy Zheng <icenowy@aosc.xyz>
---
 drivers/gpio/sunxi_gpio.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpio/sunxi_gpio.c b/drivers/gpio/sunxi_gpio.c
index b47cc66c58..7be9e25cd7 100644
--- a/drivers/gpio/sunxi_gpio.c
+++ b/drivers/gpio/sunxi_gpio.c
@@ -354,11 +354,13 @@ static const struct udevice_id sunxi_gpio_ids[] = {
 	ID("allwinner,sun8i-h3-pinctrl",	a_all),
 	ID("allwinner,sun8i-r40-pinctrl",	a_all),
 	ID("allwinner,sun9i-a80-pinctrl",	a_all),
+	ID("allwinner,sun50i-a64-pinctrl",	a_all),
 	ID("allwinner,sun6i-a31-r-pinctrl",	l_2),
 	ID("allwinner,sun8i-a23-r-pinctrl",	l_1),
 	ID("allwinner,sun8i-a83t-r-pinctrl",	l_1),
 	ID("allwinner,sun8i-h3-r-pinctrl",	l_1),
 	ID("allwinner,sun9i-a80-r-pinctrl",	l_3),
+	ID("allwinner,sun50i-a64-r-pinctrl",	l_1),
 	{ }
 };
 
-- 
2.13.6

